soccer/MainWindow.cpp

```diff
@@ -1302,6 +1305,7 @@ void MainWindow::on_logPlaybackPrevFrame_clicked() {
     _doubleFrameNumber -= 1;
 }
 
+// NOTE:Pause should possibly be locked
```

Comment:
I don't think pause needs to be locked. `bool` is an atomic type, so there's really no reason that would be necessary.